discoverecs.py

```diff
@@ -457,11 +461,11 @@ def task_info_to_targets(task_info):
 
 
 class Main:
-    def __init__(self, directory, interval, default_scrape_interval_prefix):
+    def __init__(self, directory, interval, default_scrape_interval_prefix, cluster):
         self.directory = directory
         self.interval = interval
         self.default_scrape_interval_prefix = default_scrape_interval_prefix
-        self.discoverer = TaskInfoDiscoverer()
+        self.discoverer = TaskInfoDiscoverer(cluster)
```

Comment:
Keyword arguments would be preferable here

```suggestion
        self.discoverer = TaskInfoDiscoverer(cluster_arns=cluster_arns)
```

Reply:
Awesome. Thanks